packages/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js

```diff
@@ -8,25 +8,435 @@
  * @format
  */
 
-import NativeAnimatedNonTurboModule from './NativeAnimatedModule';
-import NativeAnimatedTurboModule from './NativeAnimatedTurboModule';
 import NativeEventEmitter from '../NativeEventEmitter';
 import Platform from '../../../exports/Platform';
-import type {EventConfig} from './AnimatedEvent';
 import type {
   EventMapping,
   AnimatedNodeConfig,
   AnimatingNodeConfig,
 } from './NativeAnimatedModule';
-import type {AnimationConfig, EndCallback} from './animations/Animation';
+import type { EndCallback} from './animations/Animation';
 import type {InterpolationConfigType} from './nodes/AnimatedInterpolation';
-import invariant from 'fbjs/lib/invariant';
+
+const MAX_SAFE_INTEGER = 9007199254740991
+const MAX_ARRAY_LENGTH = 4294967295
 
 // TODO T69437152 @petetheheat - Delete this fork when Fabric ships to 100%.
-const NativeAnimatedModule =
+/* const NativeAnimatedModule =
   Platform.OS === 'ios' && global.RN$Bridgeless
     ? NativeAnimatedTurboModule
-    : NativeAnimatedNonTurboModule;
+    : NativeAnimatedNonTurboModule;*/
```

Comment:
The style of this file seems to be to grab a interface-compatible `NativeAnimatedModule` from imports. So I'd expect to see an if-statement for web here that imports the code you added below.

Since this is a vendored file, it will help enable updating this from RN itself if we follow the same style. In short, minimize changes within the /vendor/ folder and import plugins from outside.